pallets/asset-manager/src/lib.rs

```diff
@@ -557,23 +561,73 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_runtime_upgrade() -> Weight {
-			let mut reads: Weight = 0;
-			let mut writes: Weight = 0;
-			LocationAssetId::<T>::iter().for_each(|(location, _asset_id)| {
```

Comment:
Is LocationAssetId continuing to be in use after the upgrade of not?

If not, please drain it and check it exists in the pre- and doesn't exist anymore in the post check

Reply:
`LocationAssetId` is in use.